mpsl/CHANGELOG.rst

```diff
@@ -16,20 +16,18 @@ Added
 =====
 
 * Added a new signal to the MPSL timeslot: ``MPSL_TIMESLOT_SIGNAL_OVERSTAYED``.
-  This signal is given to the application when a timeslot session is closed too late. (DRGN-14677)
+  This signal is given to the application when a timeslot session is closed too late (DRGN-14677).
 
-* Added a new clock configuration option :c:member:`skip_wait_lfclk_started` in :c:struct:`mpsl_clock_lfclk_cfg_t`,
-  which does not wait for the start of Low Frequency Clock. (DRGN-14204)
+* Added a new clock configuration option :c:member:`skip_wait_lfclk_started` in :c:struct:`mpsl_clock_lfclk_cfg_t`, which does not wait for the start of the low-Frequency clock (DRGN-14204).
 
-* Added macro MPSL_RESERVED_PPI_CHANNELS for a bit mask of (D)PPI channels
-  reserved by MPSL (DRGN-13356).
+* Added the macro ``MPSL_RESERVED_PPI_CHANNELS`` for a bitmask of (D)PPI channels reserved by MPSL (DRGN-13356).
 
 Bugfixes
 ========
 
-* Fixed an issue where Low Frequency Clock was configured incorrectly
-  when the source configuration signal was set to either External Full swing or External Low swing. (DRGN-15064)
+* Fixed an issue where the low-frequency clock was configured incorrectly when the source configuration signal was set to either *external full swing* or *external low swing* (DRGN-15064).
 
+* Fixed an issue where MPSL waited for the low-frequency clock to start even when it was configured to not wait for it (DRGN-15176).
```

Comment:
This needs manifest update.
See my 2 PRs: 
https://github.com/nrfconnect/sdk-nrfxlib/pull/372
https://github.com/nrfconnect/sdk-nrf/pull/3716

Reply:
Why is there 2 PRs (378, 372) addressing the same problem?

Reply:
This one should not address the issue from 372. It pops up here since it is automatically created PR and it picks up unmerged changes from 372.

Reply:
So we should finish 372 before this one then?

Reply:
Yes, we have to

Reply:
Created manifest PR here: https://github.com/nrfconnect/sdk-nrf/pull/3747

Reply:
PR 372 and corresponding change to nrf-sdk are merged now. You need to rebase.